src/app/product-details/product-details.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { Product } from '../model/product';
+
+@Component({
+  selector: 'app-product-details',
+  templateUrl: './product-details.component.html',
+  styleUrls: ['./product-details.component.scss']
+})
+export class ProductDetailsComponent {
+  @Input() product!: Product;
+  @Output() productUpdated: EventEmitter<Product> = new EventEmitter<Product>();
+
+  productCategories: string[] = ['Personal Care', 'Food/Drink', 'Lawn & Garden'];
```

Comment:
In your next iteration, will you make this an input as well like the products? Sort of mimicking how the data would flow if the options were retrieved from the server.

Reply:
Will do!